app/views/users/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>Here is a list of all Users</h1>
+<div class="user-container">
+  <% @users.each do |user| %>
+    <div class="user-wrapper">
+        <%= link_to image_tag(user.photo), user %>
+        <div class="user-details">
+        <h2><%= user.name %></h2>
+        <h5>Number of posts: <%= user.posts_counter %></h5>
+        </div>
+    </div>
```

Comment:
- [ ] [OPTIONAL] Please consider using partials as they make your code look cleaner. You can have a file `app/views/users/_user.html.erb` and simply do `<%= render @users %>` 👍